doc/nrf/known_issues.rst

```diff
@@ -979,6 +979,50 @@ KRKNWK-8133: CSMA-CA issues
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15695: Auxiliary advertising channel packets may be sent on a channel that is not in the channel map.
+  The implementation in these versions is replaced with proper implementations of Channel Selection Algorithms #1 and #2 in newer |NCS| versions.
+
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15469: Slave connections can disconnect prematurely if there are scheduling conflicts with other roles.
+  This affects all SoftDevices.
+
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-13594: The channel map provided by the LE Host Set Channel Classification HCI command is not always applied on the secondary advertising channels.
+  In this case, the extended advertiser can send secondary advertising packets on channels which are disabled by the Host.
+  This can result in a scanner that is unable to receive some advertising packets.
+
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```

Comment:
Zephyr Write TX Power Level was added in NCS 1.3.0, so it does not affect earlier revisions.
```suggestion
.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
```